client/blocks/props-viewer/test/index.jsx

```diff
@@ -0,0 +1,49 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+import PropsViewer from '../index';
+
+const components = require( '../../../../server/devdocs/proptypes-index.json' );
+
+const findRealComponent = ( slug ) => {
```

Comment:
My personal strategy with duplicate code is to always duplicate once before refactoring for reuse. I do need to address this.